AdobeStockImageAdminUi/Test/Mftf/Test/AdminUserSeesKeywordsInTheImagePreviewTest.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminUserSeesKeywordsInTheImagePreviewTest">
+        <annotations>
+            <stories value="Admin User Sees keywords in the image preview "/>
+            <title value="Admin should be able to see keywords in the image preview"/>
+            <description value="Admin should be able to able to see keywords in the image preview"/>
+            <severity value="CRITICAL"/>
+            <group value="AdobeStockIntegration"/>
+        </annotations>
+        <before>
```

Comment:
Please use the following before and after nodes:

        <before>
            <actionGroup ref="LoginActionGroup" stepKey="LoginAsAdmin"/>
            <actionGroup ref="AdminAdobeStockSetConfigActionGroup" stepKey="setCorrectModuleConfig"/>
        </before>
        <after>
            <actionGroup ref="resetAdminDataGridToDefaultView" stepKey="resetAdminDataGridToDefaultView"/>
            <actionGroup ref="logout" stepKey="logout"/>
        </after>

Reply:
ok